_data-prepper/common-use-cases/log-analytics.md

```diff
@@ -11,15 +11,15 @@ Data Prepper is an extendable, configurable, and scalable solution for log inges
 
 The following image shows all of the components used for log analytics with Fluent Bit, Data Prepper, and OpenSearch.
 
-![Log analytics component]({{site.url}}{{site.baseurl}}/images/data-prepper/log-analytics/log-analytics-components.png)
+![Log analytics component]({{site.url}}{{site.baseurl}}/images/data-prepper/log-analytics/log-analytics-components.jpg)
 
 In the application environment, run Fluent Bit. Fluent Bit can be containerized through Kubernetes, Docker, or Amazon Elastic Container Service (Amazon ECS). You can also run Fluent Bit as an agent on Amazon Elastic Compute Cloud (Amazon EC2). Configure the [Fluent Bit http output plugin](https://docs.fluentbit.io/manual/pipeline/outputs/http) to export log data to Data Prepper. Then deploy Data Prepper as an intermediate component and configure it to send the enriched log data to your OpenSearch cluster. From there, use OpenSearch Dashboards to perform more intensive visualization and analysis. 
```

Comment:
I know I'm not obligated to review this, but I just wonder if we can start this sentence with some kind of intro to what they are doing? Like, "To get started using log analytics in Data Prepper, run Fluent Bit in the application environment of your choice. ...  Also, the sentence with "Then..." should have a better starter phrase, like, "After configuring Fluent Bit, deploy Data Prepper..." (The paragraph seems a bit rushed.)